demo/src/app/components/rating/demos/decimal/rating-decimal.ts

```diff
@@ -29,4 +29,8 @@ import { NgbRatingModule } from '@ng-bootstrap/ng-bootstrap';
 })
 export class NgbdRatingDecimal {
 	currentRate = 3.14;
+
+	ariaText(r: NgbRating) {
+		return `${r.nextRate} out of ${r.max} hearts`;
```

Comment:
`nextRate` is not a public API, so we shouldn't show it in demos. That's also why I', not a fan of passing `NgbRating`